solidity/contracts/Peggy.sol

```diff
@@ -57,12 +57,13 @@ uint256 public state_powerThreshold;
     address indexed _token,
     uint256 _eventNonce
   );
-  event SendToCosmosEvent(
+  event sendToInjectiveEvent(
```

Comment:
capitalization @mankenavenkatesh

Reply:
Also, as mentioned in the spec, we should preserve the existing `event SendToCosmosEvent` definition @mankenavenkatesh